packages/manager/apps/pci-savings-plan/src/types/savingsPlanConsumption.type.ts

```diff
@@ -1,33 +1,40 @@
 import { z } from 'zod';
 
+const moneySchema = z.object({
+  currencyCode: z.string(),
+  priceInUcents: z.number().nullable(),
+  text: z.string(),
+  value: z.number(),
```

Comment:
I am not sure to understand the goal of this, it seems that there a lot of values which are all related to the same thing (text, value, priceInUCents)